LICENSE

```diff
@@ -8,11 +8,8 @@ and "the Examples" refers to all files marked "Copyright CNRS
 Université Paris Diderot" in directory "examples" and its
 sub-directories.
 
-The Compiler and Library are distributed under the terms of the GNU
-Lesser General Public License version 2 (included below).
-
-The Examples are distributed under the terms of the GNU Lesser
-General Public License version 2 (included below).
+Both the Compiler and Library and the Examples are distributed under the
```

Comment:
Both A and B and C reads weird to me.

Reply:
I pushed another version: `The Compiler and Library, as well as the Examples, are distributed ..."

Reply:
LGTM